app/cloudinit/user_data/ubuntu_linux

```diff
@@ -267,9 +267,10 @@ runcmd:
   # Download and install auto-injection if needed
   {{ if .AutoInjectionBinaryURI }}
   - |
-    wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 -nv "{{ .AutoInjectionBinaryURI }}/{{ .Platform.OS }}/{{ .Platform.Arch }}/auto-injection" -O /usr/bin/auto-injection || \
-    wget --retry-connrefused --tries=10 --waitretry=10 -nv "{{ .AutoInjectionBinaryURI }}/{{ .Platform.OS }}/{{ .Platform.Arch }}/auto-injection" -O /usr/bin/auto-injection
-  - chmod 777 /usr/bin/auto-injection
+    mkdir -p /tmp/harness/bin
+    wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 -nv "{{ .AutoInjectionBinaryURI }}/{{ .Platform.OS }}/{{ .Platform.Arch }}/auto-injection" -O /tmp/harness/bin/auto-injection || \
+    wget --retry-connrefused --tries=10 --waitretry=10 -nv "{{ .AutoInjectionBinaryURI }}/{{ .Platform.OS }}/{{ .Platform.Arch }}/auto-injection" -O /tmp/harness/bin/auto-injection
+  - chmod 777 /tmp/harness/bin/auto-injection
```

Comment:
This will be backward incompatible. CI manager would still be sending `/usr/bin/auto-injection` path. Also in case of rollback of any service (ci-manager or runner) this will break.

Reply:
we are adding both the paths old and new from ci-manager so it remains backward compatible